docs/secure/user-auth.md

```diff
@@ -1,43 +1,36 @@
 # User authentication
 
 Percona Everest provides user authentication to ensure that only authorized users can access the databases hosted on Percona Everest. Securing access to these databases is crucial to safeguard sensitive data and restrict database access to authorized users only. 
-
-To access Percona Everest:
 {.power-number}
 
-1. Copy the authorization token that is displayed on the terminal while [installing Percona Everest](../install/installEverest.md).
+1. When you [install Percona Everest](../install/installEverest.md), an `admin` user is automatically created. To retrieve the password for the `admin` user, run the following command:
 
-    ![!image](../images/everest_user-auth-token.png)
+    ```sh
+    everestctl accounts initial-admin-password
+    ```
 
-2. Navigate to the Everest UI.
+    We strongly recommend that you reset the  initial `admin` password after installation.
```

Comment:
Missing a link to the docs on how to do that which is actually further down in this page...